coralproject/talk
# [CORL-1499] Cleanly dipose pym listeners

## What does this PR do?
- Cleanly dipose pym listeners

## How do I test this PR?
- Before this fix, the following scenario would case an error:
  - Click on share to open popup
  - click outside iframe to cause an _ClickOutside_ event that causes the popup to close
  - Notice the error
- After this fix the error will no longer occur
